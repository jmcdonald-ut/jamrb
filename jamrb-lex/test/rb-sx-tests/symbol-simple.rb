:foo
:foo_bar
a = :fO12ffff____xcf
