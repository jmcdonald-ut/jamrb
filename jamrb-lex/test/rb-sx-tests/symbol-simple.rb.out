((Token ((1 0) on_symbeg ":"))
 (Token ((1 1) on_ident "foo"))
 (Token ((1 4) on_nl "\n"))
 (Token ((2 0) on_symbeg ":"))
 (Token ((2 1) on_ident "foo_bar"))
 (Token ((2 8) on_nl "\n"))
 (Token ((3 0) on_ident "a"))
 (Token ((3 1) on_sp " "))
 (Token ((3 2) on_op "="))
 (Token ((3 3) on_sp " "))
 (Token ((3 4) on_symbeg ":"))
 (Token ((3 5) on_ident "fO12ffff____xcf"))
 (Token ((3 20) on_nl "\n")))
